### PR TITLE
(WIP) Create team-members database & implement ducks/sagas

### PR DESCRIPTION
**Pull request by naomiaz**
- Add team-members to JSON database
- Create GET, POST, DELETE calls using sagas
- Update CSS: improve BEM naming, separate existing components (MainNav, ProfileButton), extend mobile media query to tablet.

### CODE REVIEW

File: components/time-entry-overview/TimeEntryOverview.jsx [hunk 1]
```diff
@@ -59,33 +59,33 @@ class TimeEntryOverview extends React.Component {
           timeEntries={timeEntries}
         />
 
-        <section className="row time-entry-overview">
+        <section className="time-entry-overview">
           {timeEntries.map((currentTimeEntry, index, array) => {
             // if (index === 0 ) { date + component } ------->> 0 is falsy
```
Comment (seantijmons) [inline components/time-entry-overview/TimeEntryOverview.jsx]: delete this comment

File: components/time-entry-overview/time-entry-overview.scss [hunk 2]
```diff
@@ -1,22 +1,23 @@
 @import "../../shared/styles/main";
 
-.time-entry-overview {
-  margin: 32px 0;
+.time-entry {
+  &-overview {
+    @include section-row;
```
Comment (seantijmons) [inline components/time-entry-overview/time-entry-overview.scss]: use the @include container for this instead

File: components/team-member-form/TeamMemberForm.jsx [hunk 3]
```diff
@@ -1,16 +1,93 @@
...
+    const { newTeamMember, validity } = this.state;
+    const {
+      firstName, lastName, email, address, zip, city, biography, socialsTwitter, socialsFacebook
+    } = newTeamMember;
+    const { isFormSaving } = this.props;
     return (
-      <section className="team-member-form row">
-        <form action="" id="newmember">
+      <section className="team-member-form">
+        <form id="newmember" ref={this.formElement} onSubmit={this.handleSubmit}>
```
[the remark this answers is not in the document]
Comment (naomiaz) [reply components/team-member-form/TeamMemberForm.jsx]: Hmm doesn't seem to work. But I'll keep it in mind and try it another time as I have to move on now :)

File: components/nav-header/nav-header.scss [hunk 4]
```diff
@@ -6,179 +6,66 @@
...
-.logo {
-  font-family: $font-secondary;
-  padding: 11px 0;
-  text-align: center;
+  @include tablet {
+    height: 100%;
+    justify-content: space-between;
+    min-height: $main-navigation-height;
+    padding: 0 30px;
+    position: static;
```
[the remark this answers is not in the document]
Comment (naomiaz) [reply components/nav-header/nav-header.scss]: Will solve this on next css feature branch

File: components/team-member-item/team-member-item.scss [hunk 5]
```diff
@@ -9,11 +9,16 @@
...
     justify-content: space-between;
     padding: 20px;
     width: 100%;
+
+    &:hover {
+      background-color: $color-monochrome-2;
+    }
   }
 
   &__primary-wrapper {
```
[the remark this answers is not in the document]
Comment (naomiaz) [reply components/team-member-item/team-member-item.scss]: Will fix on next css feature branch

File: components/time-entry-overview/TimeEntryOverview.jsx (hunk 1 above)
Comment (naomiaz) [reply components/time-entry-overview/TimeEntryOverview.jsx]: But it's not commented out code :( it explains the map function

**Pull request merged**